R/xml_find.R

```diff
@@ -203,3 +207,13 @@ xml_find_lgl.xml_nodeset <- function(x, xpath, ns = xml_ns(x)) {
 xml_find_lgl.xml_missing <- function(x, xpath, ns = xml_ns(x)) {
    logical(0)
 }
+
+# Deprecated functions ----------------------------------------------------
+
+#' @rdname xml_find_all
+#' @usage NULL
+#' @export
+xml_find_one <- function(x, xpath, ns = xml_ns(x)) {
+  .Deprecated("xml_find_first")
```

Comment:
I'll run a revdep_check() now and see how many packages generate warnings, if it is a lot we can just use a `message()` instead of `.Deprecated()`.